requirements.in

```diff
@@ -1,6 +1,6 @@
 connexion
 flask_restx
-funding-service-design-utils>=1.0.21,<1.1.0
+funding-service-design-utils>=1.0.22,<1.1.0
```

Comment:
For future reference it is possible to do this without updating the `.in` file by upgrading the individual dependency e.g.
```
pip-compile --upgrade-package funding-service-design-utils requirements.in && pip-compile --upgrade-package funding-service-design-utils requirements-dev.in
```